index.js

```diff
@@ -209,14 +210,26 @@ function fastifyView (fastify, opts, next) {
 
           partialsHtml[key] = data
           if (--filesToLoad === 0) {
-            lru.set(`${page}-Partials`, partialsHtml)
+            lru.set(cacheKey, partialsHtml)
             callback(error, partialsHtml)
           }
         })
       })
     }
   }
 
+  function getPartialsCacheKey (page, partials, requestedPath) {
+    let cacheKey = page
+
+    Object.keys(partials).forEach(function (key) {
+      cacheKey += `|${key}:${partials[key]}`
+    })
```

Comment:
Could you please use a for..of loop instead?

Reply:
I've pushed a `fixup` commit that uses a `for..of` instead.

What is the motivation to prefer that over `forEach` in this case?